robottelo/ui/base.py

```diff
@@ -122,35 +118,21 @@ def handle_alert(self, really):
             alert.dismiss()
 
     def select_deselect_entity(self, filter_key, loc, entity_list):
-        """
-        Function to select and deselect entity like OS, Partition Table, Arch
-        from selection list or by selecting relevant checkbox.
+        """Function to select and deselect entity like OS, Partition Table,
+        Arch from selection list or by selecting relevant checkbox.
+
         """
         for entity in entity_list:
-            strategy, value = common_locators["filter"]
+            strategy, value = common_locators['filter']
             # Scroll to top
             self.browser.execute_script('window.scroll(0, 0)')
             txt_field = self.wait_until_element((strategy, value % filter_key))
             if txt_field:
                 txt_field.clear()
                 txt_field.send_keys(entity)
-                strategy, value = loc
-                select_element = self.wait_until_element((strategy,
-                                                          value % entity))
-                if select_element:
-                    select_element.click()
-                else:
-                    raise UINoSuchElementError(
-                        "Couldn't find element from selection list")
+                self.click(loc)
```

Comment:
I am not sure whether that statement is correct. Maybe it should be
```self.click(strategy, value % entity)```
instead
```self.click(loc)```
I don't know why we need that ```loc``` variable at all